src/main/java/frc/robot/Constants.java

```diff
@@ -144,6 +144,8 @@ public static final class AutoConstants { //TODO: The below constants are used i
     /* Constraint for the motion profilied robot angle controller */
     public static final TrapezoidProfile.Constraints kThetaControllerConstraints = new TrapezoidProfile.Constraints(
         kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
+
+    public static final PathConstraints maxVelocity = (4, 3);
```

Comment:
could you make velocity and accerlation separate?

for example,
public static final velocity = 4;
public static final acceleration = 3;